integration_test/readme.md

```diff
@@ -17,9 +17,13 @@ This will start two containers `amoc-1` and `amoc-2` and will wait until healthc
 
  `./integration_test/test_distribute_scenario.sh`
  
-This will install sample scenario `sample_test.erl` on node `amoc-1` using curl and then verify that scenario was propagated to the other node in the cluster `amoc-2`.
+This will install sample scenario `dummy_scenario.erl` on node `amoc-1` using curl and then verify that scenario was propagated to the other node in the cluster `amoc-2`.
```

Comment:
```suggestion
This will install a sample `dummy_scenario.erl` on node `amoc-1` using curl and then verify that it was propagated to the other node in the cluster `amoc-2`.
```